.github/workflows/build.yml

```diff
@@ -3,18 +3,46 @@ name: build
 
 on:
   push:
+    branches:
+      - "**"
+    tags:
+      - "v*.*.*"
   pull_request:
+  schedule:
+    - cron: "0 10 * * *"  # everyday at 10am
   repository_dispatch:
+    # Respond to rebuild requests. See: https://github.com/cisagov/action-apb/
     types: [apb]
+  workflow_dispatch:
+    inputs:
+      remote-shell:
+        description: "Debug with remote shell"
+        required: true
+        default: "false"
+      image-tag:
+        description: "Tag to apply to pushed images"
+        required: true
+        default: "dispatch"
 
 env:
+  BUILDX_CACHE_DIR: ~/.cache/buildx
   CURL_CACHE_DIR: ~/.cache/curl
+  IMAGE_NAME: cisagov/code-gov-update
   PIP_CACHE_DIR: ~/.cache/pip
+  # The cryptography package for Python only explicitly supports 64-bit
+  # platforms. I observed issues with installation on all of the 32-bit
+  # platforms otherwise available for the base image we use. As a result
+  # the following platforms cannot be supported at present: linux/386,
+  # linux/arm/v5, and linux/arm/v7
```

Comment:
This PR is an intermediate one just to get this on the appropriate skeleton. I plan on following this PR up with one to update the actual image configuration which includes using a newer base image. I was planning on making an issue for anything that cropped up with the newer base image and only documenting in code issues with the current image. Since it's a two-part total PR I'm using an intermediate branch (`improvement/modernize_project`) for this PR and the one to be that I mentioned. That is all to say do you mind if I wait and make any such issues around the image we'll be using (I'm planning on switching to `python:3.10.5-slim-bullseye`)?

Reply:
OK, waiting until the next PR is fine by me.